src/components/VMediaInfo/VCopyLicense.vue

```diff
@@ -13,20 +13,22 @@
           {{ $t(`media-details.reuse.copy-license.${tab}`) }}
         </VTab>
       </template>
-      <VLicenseTabPanel :tab="tabs[0]">
-        <!-- Disable reason: We control the attribution HTML generation so this is safe and will not lead to XSS attacks -->
-        <!-- eslint-disable vue/no-v-html -->
-        <div v-html="getAttributionMarkup({ includeIcons: false })" />
-        <!-- eslint-enable vue/no-v-html -->
-      </VLicenseTabPanel>
-      <VLicenseTabPanel :tab="tabs[1]">
-        <p id="attribution-html" class="font-mono break-all" dir="ltr">
-          {{ getAttributionMarkup() }}
-        </p>
-      </VLicenseTabPanel>
-      <VLicenseTabPanel :tab="tabs[2]">
-        {{ getAttributionMarkup({ isPlaintext: true }) }}
-      </VLicenseTabPanel>
+      <template #default="{ selectedTabId }">
+        <VLicenseTabPanel :tab="tabs[0]" :selected="selectedTabId === 0">
```

Comment:
Given tabs have an ID already (the value passed to `tab` prop) why do we use the index? It seems potentially flaky if a Tab component fully dismounts and re-renders, re-adding itself to the end of the list but still logically "first" within the DOM content :thinking: Using the stable ID would be a little more reliable if that is the case.